glide-core/src/client/value_conversion.rs

```diff
@@ -221,6 +237,7 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
         b"HEXISTS" | b"HSETNX" | b"EXPIRE" | b"EXPIREAT" | b"PEXPIRE" | b"PEXPIREAT"
         | b"SISMEMBER" | b"PERSIST" => Some(ExpectedReturnType::Boolean),
         b"SMEMBERS" => Some(ExpectedReturnType::Set),
+        b"SMISMEMBER" => Some(ExpectedReturnType::ArrayOfBools),
```

Comment:
Is Bool and Boolean the same in Rust?

Reply:
![image](https://github.com/Bit-Quill/glide-for-redis/assets/88679692/1638df94-90b8-439b-9dac-35552416aa88)

Reply:
rust has bool, not boolean